### ISSUE TITLE
Not loading cram

### ISSUE THREAD

**Issue by taniagmangolini** [GMOD/cram-js#135]
Recently I have opened an issue on IGV repository: https://github.com/igvteam/igv.js/issues/1805 about a problem loading cram files. Initially, the problem was on the IGV tool side because there was a harded-coded limit related to the fetch size limit. They removed the limit, but now the cram file does not load. IGV developers recommended openning an issue here to report the problem. It seems that cram-js cannot load deep coverage cram files. I can send you the files I am trying to load, if necessary.

**Comment by cmdcolin** [comment]
yes if you can send the cram files and reference genome fasta that would be great, you can send to colin.diesh@gmail.com 

If there are any error messages that go with it let me know as well. Sometimes there are just memory and performance challenges wirh e.g. the deep coverage filesbut we try to handle as best we can :)

**Comment by taniagmangolini** [comment]
The IGV tries to load and after some time it shows the following error: [https://private-user-images.githubusercontent.com/25271048/330128500-a42b448b-2f0e-4554-a359-7ec4304ea4dd.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTYyMjkxODQsIm5iZiI6MTcxNjIyODg4NCwicGF0aCI6Ii8yNTI3MTA0OC8zMzAxMjg1MDAtYTQyYjQ0OGItMmYwZS00NTU0LWEzNTktN2VjNDMwNGVhNGRkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MjAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTIwVDE4MTQ0NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWM1NmI5NmIxNTU4MjA5MTg0ZjgyZWE5NWY2ZDVlNjZlNDA3ZTcxMjBjZTc2NGE3NDYwYmVjYzZlNzQ4YzY5MjkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.cROCRYOcaqpWNb9i8Li0PavReRNePsDrGGiLXEnZgKA](image)

**Comment by cmdcolin** [comment]
thanks for the email. i found it loaded in the jbrowse 2, though it did have high memory consumption (which can put a page on the verge of the ah nao! type errors). i would have to do some memory profiling to see if there are any areas of memory usage that could be improved
